modin/core/dataframe/pandas/dataframe/dataframe.py

```diff
@@ -19,8 +19,8 @@
 """
 from collections import OrderedDict
 import numpy as np
-import pandas
 import datetime
+import pandas
```

Comment:
Move the pandas import back to its old position?

Reply:
Fixed!